services/lambda/api/Dockerfile

```diff
@@ -1,6 +1,11 @@
 FROM public.ecr.aws/lambda/python:3.8
```

Comment:
I needed them to test the streamlit interface. Remove them would break the streamlit service, I suggest solve any conflicts when the separate PR is rebased and eventually merged.